src/utils/embedding_store.py

```
@@ -0,0 +1,425 @@
+                        db,
+                        workspace_name=self.workspace_name,
+                        peer_name=self.peer_name,
+                        collection_name=self.collection_name,
+                        query=query,
+                        max_distance=max_distance,
+                        top_k=top_k,
+                    )
+                    db.expunge_all()
+                    return list(documents)
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)